src/views/Project/index.jsx

```diff
@@ -125,11 +124,26 @@ const tagReposMapping = repositories =>
   spinner: {
     marginTop: 3 * theme.spacing.unit,
   },
+  projectSummary: {
+    minHeight: 50,
+    padding: '15px',
+  },
+  body: {
+    padding: '15px',
+  },
+  toolbar: {
+    padding: 10,
+  },
+  headline: {
+    flex: 1,
+    whiteSpace: 'nowrap',
+  },
```

Comment:
You will probably remove many of these styling once we move to using the `Card` component. See my other comment about it.